less/components/currency-dropdown.less

```diff
@@ -3,6 +3,10 @@
   text-transform: uppercase;
   text-align: center;
   padding: 0 20px;
+  position: absolute;
+  width: 125px;
+  right: 5px;
+  top: 0;
```

Comment:
Mind giving this a try?:

1. pull `CurrencyDropdown` out of the span
2. put `CurrencyDropdown` beside, not inside, the donate now `h3`
3. add a `donate-now-header` to the `h3`
4. add `display: inline-block; width: 220px;` to `.donate-now-header` 
5. add `display: inline-block; width: 120px; vertical-align: top;` to `.currency-dropdown` 

Thoughts? It means you could remove a float right, an absolute position, make it better on screan readers, and possibly not need to add `main-page`?